src/ucp/core/ucp_ep.c

```diff
@@ -908,8 +904,12 @@ static size_t ucp_ep_config_calc_rndv_thresh(ucp_worker_t *worker,
     eager_rsc_index  = config->key.lanes[eager_lanes[0]].rsc_index;
     eager_iface_attr = ucp_worker_iface_get_attr(worker, eager_rsc_index);
 
+    /* RTS/RTR latency is used from lanes[0] */
+    rts_latency      = ucp_tl_iface_latency(context, eager_iface_attr);
+
     numerator = diff_percent * (rndv.reg_overhead * (1 + recv_reg_cost) +
-                                4 * eager_zcopy.latency + 3 * eager_zcopy.overhead) -
+                                (4 * rts_latency) +
```

Comment:
should be 2*rts_latency + 2*rndv_latency

Reply:
why?
latency is actual for RTS/RTR only, but all such communications runs over same lane

Reply:
2* latency is rts, the other 2 is for get_zcopy

Reply:
ok, done